Source/Scene/ModelExperimental/PntsLoader.js

```diff
@@ -0,0 +1,532 @@
+import AttributeCompression from "../../Core/AttributeCompression.js";
+import Cartesian3 from "../../Core/Cartesian3.js";
+import Cartesian4 from "../../Core/Cartesian4.js";
+import Check from "../../Core/Check.js";
+import ComponentDatatype from "../../Core/ComponentDatatype.js";
+import defaultValue from "../../Core/defaultValue.js";
+import defined from "../../Core/defined.js";
+import Matrix4 from "../../Core/Matrix4.js";
+import PrimitiveType from "../../Core/PrimitiveType.js";
+import MersenneTwister from "../../ThirdParty/mersenne-twister.js";
+import when from "../../ThirdParty/when.js";
+import Buffer from "../../Renderer/Buffer.js";
+import BufferUsage from "../../Renderer/BufferUsage.js";
+import AttributeType from "../AttributeType.js";
+import Axis from "../Axis.js";
+import parseBatchTable from "../parseBatchTable.js";
+import DracoLoader from "../DracoLoader.js";
+import FeatureMetadata from "../FeatureMetadata.js";
+import ResourceLoader from "../ResourceLoader.js";
+import MetadataClass from "../MetadataClass.js";
+import ModelComponents from "../ModelComponents.js";
+import PntsParser from "../PntsParser.js";
+import PropertyTable from "../PropertyTable.js";
+import ResourceLoaderState from "../ResourceLoaderState.js";
+import VertexAttributeSemantic from "../VertexAttributeSemantic.js";
+
+var Components = ModelComponents.Components;
+var Scene = ModelComponents.Scene;
+var Node = ModelComponents.Node;
+var Primitive = ModelComponents.Primitive;
+var Attribute = ModelComponents.Attribute;
+var Quantization = ModelComponents.Quantization;
+var FeatureIdAttribute = ModelComponents.FeatureIdAttribute;
+var Material = ModelComponents.Material;
+var MetallicRoughness = ModelComponents.MetallicRoughness;
+
+export default function PntsLoader(options) {
+  options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+  var arrayBuffer = options.arrayBuffer;
+  var byteOffset = defaultValue(options.byteOffset, 0);
+
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.object("options.arrayBuffer", arrayBuffer);
+  //>>includeEnd('debug');
+
+  this._arrayBuffer = arrayBuffer;
+  this._byteOffset = byteOffset;
+
+  this._parsedContent = undefined;
```

Comment:
`this._parsedContent` should be set back to undefined once it is no longer needed to help the garbage collector free some large typed arrays.